server/src/main/java/org/elasticsearch/common/logging/Loggers.java

```diff
@@ -102,7 +102,7 @@ public static Logger getLogger(String s) {
      */
     @Deprecated
     public static Logger getLogger(Class<?> clazz) {
-        return ESLoggerFactory.getLogger(clazz);
+        return ESLoggerFactory.getLogger(null, clazz);
```

Comment:
`null` looked weird, but then I saw the comment on ESLoggerFactory that this will be cleaned up at some point, so it looks okay.